### PR TITLE
feat: switched to base image

### PR DESCRIPTION
**Pull request by demeyerthom**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    debian-keyring \
+    debian-archive-keyring \
+    apt-transport-https && \
+    curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg && \
+    curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | tee /etc/apt/sources.list.d/caddy-stable.list && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+    wget \
+    supervisor \
+    caddy \
```
Comment (mvantellingen) [inline Dockerfile]: I dont think we need these packages (wget, supervisor, caddy) in the base image

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Combine RUN instructions and clean up after installations
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    debian-keyring \
+    debian-archive-keyring \
+    apt-transport-https && \
+    curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg && \
+    curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | tee /etc/apt/sources.list.d/caddy-stable.list && \
```
Comment (mvantellingen) [inline Dockerfile]: both should be removd (see next comment)

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Add user so that we run as non-root
+RUN groupadd app && useradd -g app app
+
+# Create writable dir
+RUN mkdir /app/ /home/app && chown app:app /app/ /home/app
+
+# Switch to app user
+USER app
```
Comment (mvantellingen) [inline Dockerfile]: Not sure if we want to switch to user in base image

**Review by mvantellingen — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (demeyerthom) [reply Dockerfile]: this was only required in the all-in-one package. I will move it back

File: Dockerfile (hunk 2 above)
Comment (demeyerthom) [reply Dockerfile]: Will remove

File: Dockerfile (hunk 3 above)
Comment (demeyerthom) [reply Dockerfile]: Remove only the USER directive, or just the whole bit?

**Comment by demeyerthom** [comment]
Removed the packages that are only used under specific circumstances, removed the user creation

**Pull request merged**